piker/clearing/_paper_engine.py

```diff
@@ -286,11 +274,14 @@ async def fake_fill(
             ledger.update(self._trade_ledger)
 
         # Write to pps toml right now
-        with open_pps(self.broker, 'piker-paper') as table: 
+        with open_pps(self.broker, 'piker-paper') as table:
```

Comment:
I realize this allows you to greedily update the `pps.toml` but it might be worth it to to assign the table instance once at startup and then enable the *immediate writing* through `PaperBoi` config of some sort?

Not a huge deal for now, but maybe something to make a `TODO:` for 😉